kitchen.recipes.yml

```diff
@@ -399,3 +399,20 @@ suites:
     attributes:
       dependencies:
         - recipe:aws-parallelcluster-install::chrony
+  - name: imds_configured
+    run_list:
+      - recipe[aws-parallelcluster::add_dependencies]
+      - recipe[aws-parallelcluster-config::imds]
+    verifier:
+      controls:
+        - imds_configured
+    attributes:
+      dependencies:
+        - recipe:aws-parallelcluster-install::directories
+        - resource:package_repos
+        - resource:install_packages
```

Comment:
why do you need the entire install_packages repo? Do you know which are the required packages?

Reply:
Yes, the `iptables` package.

Reply:
if you need iptables, this has been added as part of IMDS feature so I think we can move outside of base_package: https://github.com/aws/aws-parallelcluster-cookbook/commit/17ee7fbd123c0e32e7b06904bc252a39cdb3ff90

Reply:
As agreed offline, moved the `iptables` dependency in the `imds` recipe.